motor/render/vulkan_renderer.cpp

```diff
@@ -68,55 +152,27 @@ vk::Instance CreateInstance()
                          "Couldn't createInstance");
 }
 
-vk::PhysicalDevice SelectPhyiscalDevice(const vk::Instance& vk_inst)
+vk::PhysicalDevice SelectPhyiscalDevice(const vk::Instance& vk_inst,
+                                        const vk::SurfaceKHR& surface)
 {
   std::vector<vk::PhysicalDevice> devices = VkSuccuessOrDie(
       vk_inst.enumeratePhysicalDevices(), "Couldn't enumeratePhysicalDevices");
   CHECK(!devices.empty()) << "No vulkan device found";
```

Comment:
again if this returns optional, no need to `CHECK` here, caller can handle the failure.